src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(value) {
+      result += value;
+    },
+    subtract(value) {
+      result -= value;
+    },
+    multiply(value) {
+      result *= value;
+    },
+    divide(value) {
+      result /= value;
+    },
+    reset(value) {
+      result = 0;
```

Comment:
The `reset` method should not accept any parameters according to the task description. It must reset the `result` to 0 without any arguments. Please remove the `value` parameter as it is unused and not needed. The method should be defined as `reset() { ... }`.